server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -114,6 +116,16 @@ fn generate_response_requisition(
         comment: None,
     };
 
+    system_log_entry(
```

Comment:
Can the logs be inside processor please. (they are less `hidden` away)

Reply:
[90faf29](https://github.com/openmsupply/open-msupply/pull/666/commits/90faf29237d024125225ff80f11c21841b978f1f)